di_website/templates/general/general_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layouts/default.html' %}
+
+{% load static wagtailcore_tags wagtailimages_tags string_utils card_tags %}
+
+{% block hero %}
+    {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+    <section class="section">
+        <div class="row row--narrow">
+            <div class="l-sidebar">
+                
+                {% include 'includes/partials/side_nav.html'%}
+
+                <div class="l-sidebar__body">
+                    <div class="is-typeset wysiwyg">
+                        {% for block in page.body %}
+                            {% include_block block %}
+                        {% endfor %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    {% include './includes/team_member_quote.html'%}
+
+    {% for content in page.content %}
+        {% include_block content %}
+    {% endfor %}
+
+   
+    {% for block in page.downloads %}
+        {% include_block block %}
+    {% endfor %}
+    
+    <section class="section">
+        {% get_other_pages calling_page=self heading=page.other_pages_heading %}
```

Comment:
Visually, this is rather similar to `blocks/banner` but I can see why you did it this way to avoid having to pick out team member photo, name, and position. Not sure if we want to refactor this just into a banner or not, but in the mean time, approve!